src/Tools/dotnet-trace/CommandLine/Commands/CollectCommand.cs

```diff
@@ -267,7 +285,14 @@ private static string GetSize(long length)
             new Option(
                 alias: "--profile",
                 description: @"A named pre-defined set of provider configurations that allows common tracing scenarios to be specified succinctly.",
-                argument: new Argument<string>(defaultValue: "runtime-basic") { Name = "profile_name" },
+                argument: new Argument<string>(defaultValue: "runtime-basic") { Name = "profile-name" },
                 isHidden: false);
+
+        private static Option DurationOption() =>
+            new Option(
+                alias: "--duration",
+                description: @"When specified, will trace for the given timespan and then automatically stop the trace. Provided in the form of dd:hh:mm:ss.",
+                argument: new Argument<TimeSpan>(defaultValue: null) { Name = "duration-time-in-seconds" },
```

Comment:
nit: Is the Name still `duration-time-in-seconds`? The description seems to suggest otherwise

Reply:
ah, good catch!